tests/unit_tests/model/test_model_parameter.py

```diff
@@ -31,7 +32,7 @@ def test_get_parameter_dict(telescope_model_lst):
 
 
 def test_get_parameter_value(telescope_model_lst):
-    tel_model = telescope_model_lst
+    tel_model = copy.deepcopy(telescope_model_lst)
```

Comment:
We usually uses copy.copy but I checked now in the reference API and deepcopy is probably better. Should we worry that `copy.copy` is not doing what we think it is in other parts of the code?

Reply:
My assumption is always that if the test does not fail, then `copy.copy` is enough. In this case, we required a deepcopy.